lib/clients/rtm/client.js

```diff
@@ -617,6 +628,35 @@ RTMClient.prototype.sendMessage = function sendMessage(text, channelId, optCb) {
   }, optCb);
 };
 
+/**
+ * Helper for updating a sent message via the 'chat.update' API call
+ * @param message {object} message The message object to be updated,
+ * see /lib/clients/web/facets/chat.js for more details
+ * @param {Function=} optCb Optional callback
+ */
+RTMClient.prototype.updateMessage = function updateMessage(message, optCb) {
+  var err;
+  var ret;
+
+  if (this.connected) {
+    ret = this._chat.update(message.ts, message.channel, message.text, message.opts, optCb);
```

Comment:
Sorry, I forget that's internal terminology. MS = Message Server = the thing you connect to with the RTM API. My question is on the `if (this.connected)` condition—you can call `chat.update` without a live RTM connection.